json/json.go

```diff
@@ -70,6 +70,11 @@ const (
 	// checking of raw messages. It should only be used if the values are
 	// known to be valid json (e.g., they were created by json.Unmarshal).
 	TrustRawMessage
+
+	// AppendNewline is a formatting flag to enable the addition of a newline
+	// in Encode (this matches the behavior of the standard encoding/json
+	// package).
+	AppendNewline
```

Comment:
Since this is used internally only and has no effect when passed to `json.Append`, how about keeping it unexported?
```suggestion
	appendNewline
```

Reply:
Sounds good to me too.